commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/mapping/serialization/from/ToExtAttribute.kt

```diff
@@ -0,0 +1,12 @@
+package ebi.ac.uk.extended.mapping.serialization.from
+
+import ebi.ac.uk.extended.model.ExtAttribute
+import ebi.ac.uk.extended.model.ExtAttributeDetail
+import ebi.ac.uk.model.Attribute
+import ebi.ac.uk.model.AttributeDetail
+
+fun Attribute.toExtAttribute() = ExtAttribute(name, value, reference, toDetails(nameAttrs), toDetails(valueAttrs))
+
+private fun toDetails(details: List<AttributeDetail>) = details.map { ExtAttributeDetail(it.name, it.value) }
+
+internal const val TO_EXT_ATTRIBUTE_EXTENSIONS = "ebi.ac.uk.extended.mapping.serialization.from.ToExtAttributeKt"
```

Comment:
This is just cosmetic bu, It looks a bit weird to me to have a constant at the bottom of the file 😅